project/js/script.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function() {
+  let $title = $("h1#title");
+  let $data = $("div#data");
+  const data = {
+    "Dog" : "https://random.dog/woof.json",
+    "Cat" : "https://aws.random.cat/meow"
+  };
+
+
+  let random = Math.floor(Math.random() * Object.keys(data).length);
+  let head = Object.keys(data)[random];
+  $title.html(head);
+
+  switch(random){
+    case 0:
```

Comment:
Love the clean structure of `switch`! 
I know it is a mock up, but you are hard coding the case 0 and 1. If something changed in `data`, then the switch conditions won't be met and the code will return nothing. 
On the other hand, if we are sure that nothing will change in data, then there might be more readable ways of hard coding, so that users don't have to guess whether 0 is dog or cat.